send_results.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+import smtplib
+import sys
+
+from pathlib import Path
+from typing import List
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+
+COMMASPACE = ", "
+
+
+class SendResults(object):
```

Comment:
No need to inherit from `object` in python3

```suggestion
class SendResults():
```